Content.Shared/Stunnable/SharedStunSystem.cs

```diff
@@ -155,14 +155,23 @@ private void OnRefreshMovespeed(EntityUid uid, SlowedDownComponent component, Re
 
         /// <summary>
         ///     Applies knockdown and stun to the entity temporarily.
+        ///     Raises GotParalyzedEvent if successful.
         /// </summary>
         public bool TryParalyze(EntityUid uid, TimeSpan time, bool refresh,
             StatusEffectsComponent? status = null)
         {
             if (!Resolve(uid, ref status, false))
                 return false;
 
-            return TryKnockdown(uid, time, refresh, status) && TryStun(uid, time, refresh, status);
+            bool paralyzed = (TryKnockdown(uid, time, refresh, status) && TryStun(uid, time, refresh, status));
+
+            if (paralyzed)
+            {
+                var ev = new GotParalyzedEvent(uid);
+                RaiseLocalEvent(uid, ev, false);
+            }
+
+            return paralyzed;
```

Comment:
this is pretty weird, i'd rather not have stuff hardcode based on whether they got paralyzed. what you should do instead is subscribe to some event that standing state system raises and check if they got knocked down for getting them off the vehicle, imo.